app/forms/schools/year2020_form.rb

```diff
@@ -5,28 +5,18 @@ class Year2020Form
     include ActiveModel::Model
     include ActiveModel::Serialization
 
-    attr_accessor :school_id, :induction_programme_choice, :core_induction_programme_id, :full_name, :email, :participants
-
-    validates :induction_programme_choice, presence: true, on: :choose_induction_programme
```

Comment:
Why has this gone?

Reply:
I think it is superfluous due to this line beneath:
```
validates :core_induction_programme_id, presence: true, on: :choose_cip
```
The validation seems to still work:

![Uploading Screenshot 2021-09-14 at 12.26.28.png…]()